industrial_extrinsic_cal/include/industrial_extrinsic_cal/conical_pose_generator.h

```diff
@@ -0,0 +1,13 @@
+#ifndef CONICAL_POSE_GENERATOR_H
+#define CONICAL_POSE_GENERATOR_H
+
+#include <Eigen/Dense>
+#include <eigen_stl_containers/eigen_stl_vector_container.h>
+
```

Comment:
Put a comment here stating that this is a prototype of the c-style function defined in conical_pose_generator.cpp